### PR TITLE
Edits to ScalePerformance

### PR DESCRIPTION
**Pull request by osimarkrogers**
Grammatical revision, active voice

### CODE REVIEW

File: V1/Design/ScalePerformance.md [hunk 1]
```diff
@@ -6,20 +6,20 @@ uid: scalePerformance
...
+The Edge Storage component integrated with the Edge Data Store is a new item in the OSIsoft software ecosystem. It is not designed to replace any existing storage technology produced by OSIsoft. The Edge Storage component is intended as a data store that is resilient and reliable but limited in duration and scope, as appropriate for an Edge software component. The storage component is configured by default to roll off data in a FIFO (first in first out) process: as new data comes in and the size of streams exceeds the configured limits, older data is purged.
 
-If data exists in the Edge Storage component that needs to be permanently retained, it should be egressed to either the PI Data Archive (using the PI Web API OMF endpoint) or to OSIsoft Cloud Services, using the OCS OMF ingress endpoint.
+If data exists in the Edge Storage component that needs to be permanently retained, you should egress it to either the PI Data Archive (using the PI Web API OMF endpoint) or to OSIsoft Cloud Services, using the OCS OMF ingress endpoint.
```
Comment (TinaUll) [inline V1/Design/ScalePerformance.md]: no "the" in front of product names except for the PI System

File: V1/Design/ScalePerformance.md [hunk 2]
```diff
@@ -6,20 +6,20 @@ uid: scalePerformance
...
 
 ## Edge Storage scale
 
-The Edge Storage component has been designed to provide an appropriate level of storage performance on small devices. For the smallest of these devices, throughput may be limited to 10s of events per second. For larger devices with faster processors, memory, and storage, this could increase to up to 3,000 events per second. The Edge Storage component's design is focused on small devices in Edge scenarios: if high throughput or large stream counts are required, OSIsoft Cloud Services or the PI Data Archive are more appropriate choices.
+The Edge Storage component provides an appropriate level of storage performance for small devices. For the smallest of these devices, throughput may be limited to tens of events per second. For larger devices with faster processors, memory and storage, this could increase to up to 3,000 events per second. The Edge Storage component's design is focused on small devices in Edge scenarios: if high throughput or large stream counts are required, OSIsoft Cloud Services or the PI Data Archive are more appropriate choices.
```
Comment (TinaUll) [inline V1/Design/ScalePerformance.md]: no "the" in front of PI Data Archive

File: V1/Design/ScalePerformance.md (hunk 1 above)
Comment (osimarkrogers) [reply V1/Design/ScalePerformance.md]: Hmmmm.....Edge Storage is just a component of Edge Data Store, not an actual product.  Rule still applies?

File: V1/Design/ScalePerformance.md (hunk 2 above)
Comment (osimarkrogers) [reply V1/Design/ScalePerformance.md]: I think you mean Edge Storage component again....does the rule apply to components as well as products?

Comment (TinaUll) [reply V1/Design/ScalePerformance.md]: No, I mean PI Data Archive :) The only time where we use an article in front of a product is our good old PI System.

File: V1/Design/ScalePerformance.md (hunk 1 above)
Comment (TinaUll) [reply V1/Design/ScalePerformance.md]: talking about PI Data Archive here too

Comment (osimarkrogers) [reply V1/Design/ScalePerformance.md]: OK, fixed.

Comment (osimarkrogers) [reply V1/Design/ScalePerformance.md]: Checking with Chris Felts about component names.

**Pull request merged**